services/core-api/app/api/mines/permits/permit_conditions/resources/permit_conditions_resource.py

```diff
@@ -96,4 +96,22 @@ def delete(self, mine_guid, permit_guid, permit_amendment_guid, permit_condition
 
         permit_condition.save()
 
+        if permit_condition.parent_permit_condition_id is not None:
+            parent = PermitConditions.find_by_permit_condition_id(
+                permit_condition.parent_permit_condition_id)
```

Comment:
Why don't we just traverse 'permit_condition.parent_condition'. Shouldn't need a whole lookup.

Reply:
.parent didn't seem accessible outside of the scope of the model itself. Maybe a get_parent method on the model might work

Reply:
PermitCondition should get a relationship to it's parent (just like relationship to it's set of children). Also, that model should be singular.